src/sentry/api/event_search.py

```diff
@@ -703,6 +703,10 @@ def to_list(value):
     # Long term these will become more complex functions but these are
     # field aliases.
     "apdex": {"result_type": "number", "aggregations": [["apdex(duration, 300)", "", "apdex"]]},
+    "error_ratio": {
+        "result_type": "number",
+        "aggregations": [["countIf(transaction_status > 0) / count()", "", "error_ratio"]],
```

Comment:
I don't think we should leak complex clickhouse functions outside of Snuba. I am not sure why we did not add a proper validation of this column (that would not allow for this). 
Also I suspect this query will fail if Snuba decides to serve it from the events table instead of the transaction table. And since we are passing the expression for clickhouse as a string Snuba cannot parse it so it cannot use it to decide which table to use.

I would advise to resolve the alias in Snuba through a query processor (which means creating one like this: https://github.com/getsentry/snuba/blob/master/snuba/query/processors/tagsmap.py. It would unpack the query find the aggregation and tweak the aggregation field)

Should there be any problem with that let's just do the same way we did for apdex. But at least that would happen only once Snuba decided how to run the query.

Also > 0 does not seem the right way to identify errors. We should do something like
```
transaction_status != SPAN_STATUS_NAME_TO_CODE["OK"]
```